python/examples/example_jax/ExampleJax.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d4d2bc5c",
+   "metadata": {},
+   "source": [
+    "The purpose of this guide is to showcase how AMICI can be combined with differentiable programming in JAX. We will do so by reimplementing the parameter transformations available in AMICI in JAX and comparing it to the native implementation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b0a66e18",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import jax\n",
+    "import jax.numpy as jnp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa587260",
+   "metadata": {},
+   "source": [
+    "# Preparation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb2fe897",
+   "metadata": {},
+   "source": [
+    "To get started we will import a model using the [petab](https://petab.readthedocs.io). To this end, we will use the [benchmark collection](https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab), which features a variety of different models. For more details about petab import, see the respective notebook petab [notebook](https://amici.readthedocs.io/en/latest/petab.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "04804185",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cloning into 'tmp/benchmark-models'...\n",
+      "remote: Enumerating objects: 336, done.\u001b[K\n",
+      "remote: Counting objects: 100% (336/336), done.\u001b[K\n",
+      "remote: Compressing objects: 100% (285/285), done.\u001b[K\n",
+      "remote: Total 336 (delta 88), reused 216 (delta 39), pack-reused 0\u001b[K\n",
+      "Receiving objects: 100% (336/336), 2.11 MiB | 7.48 MiB/s, done.\n",
+      "Resolving deltas: 100% (88/88), done.\n"
+     ]
+    }
+   ],
+   "source": [
+    "!git clone --depth 1 https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab.git tmp/benchmark-models || (cd tmp/benchmark-models && git pull)\n",
+    "from pathlib import Path\n",
+    "folder_base = Path('.') / \"tmp\" / \"benchmark-models\" / \"Benchmark-Models\""
```

Comment:
For an individual model, you can also use the URL of the yaml file in Problem.from_yaml to just download that problem.